### PR TITLE
Some IOOS recipes

### PR DESCRIPTION
**Pull request by ocefpaf**
These recipes are simple and pure python. It should be OK to send them in a batch together.

### CODE REVIEW

**Comment by jakirkham** [comment]
LGTM. Thanks @ocefpaf.

File: recipes/geojsonio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    fn: geojsonio-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/g/geojsonio/geojsonio-{{ version }}.tar.gz
+    md5: e75b0ec0786d456dc630f93874106bfd
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    entry_points:
+        - geojsonio = geojsonio:main
+
```
Comment (pelson) [inline recipes/geojsonio/meta.yaml]: Should I add a whitespace constraint to the linter do you think?

Comment (ocefpaf) [inline recipes/geojsonio/meta.yaml]: Most pure python packages recipes, that are on PyPI, will start from `conda-skeleton`, and `conda-skeleton` did add that space in this case. Not sure if that is always true, but if it is we might want to enforce that to keep consistency. (Although I don't have any strong feelings about it.)

Comment (pelson) [inline recipes/geojsonio/meta.yaml]: Let's stay light touch 😄

**Comment by ocefpaf** [comment]
Thanks for merging @jakirkham!

**Pull request merged**